test/e2e/util.go

```diff
@@ -167,10 +183,12 @@ func waitForPodsRunningReady(ns string, minPods int, timeout time.Duration) erro
 			Logf("Error getting pods in namespace '%s': %v", ns, err)
 			continue
 		}
-		nOk := 0
+		nOk, badPods := 0, []api.Pod{}
 		for _, pod := range podList.Items {
 			if res, err := podRunningReady(&pod); res && err == nil {
 				nOk++
+			} else {
+				badPods = append(badPods, pod)
 			}
 		}
 		Logf("%d / %d pods in namespace '%s' are running and ready (%v elapsed)",
```

Comment:
I'm mixed. So the nice thing about printing durations with just `%v` is that if you do that everywhere, you never have to worry about switching your printing around when things speed up and slow down (i.e. converting between minutes, seconds, and milliseconds). The duration prints itself with the relevant units, and you go about your day. I was cleaning up some e2e tests to do this instead of manual formatting.

BUT you're right that it would be more readable. Hmmmm. Classic tradeoff.

OK, I switched it to round to seconds.